### PR TITLE
Task 1: Login

### PR DESCRIPTION
**Pull request by tulip1749**
implement login feature

### CODE REVIEW

File: PageObjects/LoginPage.js
Comment (srinikethag) [inline PageObjects/LoginPage.js:1]: URL hardcoded. Should keep all the data separate from implementation.

**Review by srinikethag — CHANGES_REQUESTED** [review]
fix the URL changes

File: PageObjects/LoginPage.js
Comment (tulip1749) [reply PageObjects/LoginPage.js:1]: To be handled in seperate task.

**Review by srinikethag — APPROVED** [review]
LGTM

**Pull request merged**